net/zmap/Portfile

```
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           cmake 1.0
```

Consider updating to the cmake 1.1 portgroup.
```suggestion
PortGroup           cmake 1.1
```